src/central_control/scripts/central_control.py

```diff
@@ -36,9 +39,15 @@ def listen(self):
     def control(self):
         print("Control loop")
 
-    def handle_steering_data(self, steering_data):
-        print("Obtained steering data")
-        self.car_controls.steering = steering_data.data
+    # Results from navigation
+    # Returns the steering angle from pure pursuit
+    # Returns the current road segment type
+    # Returns a warning of a new road segment if one is within 5 meter of the car
+    def handle_navigation_data(self, navigation_data):
```

Comment:
Not urgent, but if you're going to push an update, add a type for navigation data

Reply:
will do